lms/lib/courseware_search/test/test_lms_filter_generator.py

```diff
@@ -44,7 +44,7 @@ def test_course_id_not_provided(self):
         """
         Tests that we get the list of IDs of courses the user is enrolled in when the course ID is null or not provided
         """
-        field_dictionary, filter_dictionary = LmsSearchFilterGenerator.generate_field_filters(user=self.user)
+        field_dictionary, filter_dictionary, _ = LmsSearchFilterGenerator.generate_field_filters(user=self.user)
```

Comment:
Are there any assertions we should do on this new 3rd return value? That 3rd value is the exclude_dictionary, right?